### DISCUSSION TITLE
リアクションを受け入れない設定の提案

### DISCUSSION THREAD

**Discussion by MattyaDaihuku** [misskey-dev/misskey#14287] [Ideas]
ノートのリアクションを受け入れたくないときに使えるが、この設定なくてもいい気がする
需要があるのか気になったので皆さんの意見を聞いてみたいです

**Comment by kakkokari-gtyih** [comment]
連合からLikeのアクティビティを受け取った際にどう処理するかを検討する必要がありそう

**Comment by anatawa12** [comment]
そもそもとしていいねのみではだめなのかな

それともいいね(Like)リアクションも受けたくないということですかね
![IMG_3557](https://github.com/user-attachments/assets/000fdc56-051d-45f6-b708-2d7f0a22654a)

**Comment by MattyaDaihuku** [comment]
いいねリアクションも受けつけない設定があるといいのかな、と思いました

**Reply by anatawa12** [reply]
いいねも受け付けないのであればLikeアクティビティに関しては破棄して良さそうですね

しかし連合先(特にmastodon等非 misskey系ですが古い misskeyを含む)においていいねやリアクションが投稿元サーバーからは見えないリモートでのみ行われることにはなると思います

**Comment by kakkokari-gtyih** [comment]
技術的な懸念点もあると思いますが、それ以前にこの機能がそもそも必要かというところも議論しておくとよさそうです（本人も一番上でその通り言っておられるように）

個人的にはユースケースがあまり思いつかない（ユーザーの反応を見れなくしたいのであれば「いいねのみ」で事足りる気がします。特にMisskeyではリアクションの数よりかは比較的そこにつく絵文字の内容が重視される気がするので）
